### ISSUE TITLE
Implement profanity filter

### ISSUE THREAD

**Issue by nshmyrev** [alphacep/vosk-api#623]

**Comment by dev2devportal** [comment]
Subscribed. Looking forward to figuring this out. 
Will also hopefully be bringing in at least 1 or 2 other people to help with figuring this out and then testing in our R&D environment before sending to the Development team for integration into the LMS.

**Comment by nshmyrev** [comment]
https://pypi.org/project/profanity-filter/ is a useful library/data

**Comment by lmhawke** [comment]
Yes, that is the one we were looking at trying to figure out how to integrate with Vosk and Jitsi (Jigasi).

**Comment by abb128** [comment]
Steam's chat filtering has a feature where you can switch between three different modes: filtering nothing, filtering only slurs and filtering all profanities.

![image](https://user-images.githubusercontent.com/65567823/127741215-e0e9cdde-2086-4c29-abe0-914c1b05ef1b.png)

I think this is an important feature to consider because in many cases people may tolerate casual swearing, but not racism and slur-dropping.

**Comment by dev2devportal** [comment]
FYI, our intern was able to get profanity filtering working. I'll see about getting his information submitted into these relevant tickets with the information on how to do it (I have to get permission from work to do so).

**Comment by dev2devportal** [comment]
Here is the implementation information on what we used: https://github.com/alphacep/vosk-server/issues/110

**Comment by dev2devportal** [comment]
We still have to migrate this into our official DEV and PRD cycle, will let folks know how that goes. If there are any bugs or user issues found as it goes through official DEV and QA.
